### ISSUE TITLE
SSL certificate problem: certificate has expired (root certificate Letzencrypt)

### ISSUE THREAD

**Issue by mamaretti** [ForkIssues/Tracker#1485]
Hi Fork-Team

I get an SSL error when trying to connect a gitlab-server with letsencrypt certificate.

fatal: unable to access 'https://....: SSL certificate problem: certificate has expired

as I know root-certificate of Letzencrypt is expired in 30th Sept.
https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

and since this time i get this error

my workaround is to disable SSL-Verify via
git config --global http.sslVerify false

can you pls check if it a problem with Fork?

Thx

**Comment by dr3amnightmare** [comment]
Let's Encrypt just had one of their root certs expire. This is something they have been warning about for 2 years now and companies chose not to move over to the new root cert. Because of this, all certs referring to the old root are now invalid. This is not a Fork problem. This would be a problem with whatever endpoint you are using not updating their certs accordingly. You kinda answered your own question when you asked this. All fork does is try to communicate via git to your endpoint. The git tool checks the cert and sees that it is properly expired and refuses to connect to an https connection with an invalid cert.

TL;DR blame the endpoint for not migrating to the new root cert, not fork

**Comment by dr3amnightmare** [comment]
@mamaretti You could also try updating git itself to see if it is holding onto and using the old cert system if you have an older version of git installed

**Comment by DanPristupov** [comment]
@dr3amnightmare thank you for your comment.

>  All fork does is try to communicate via git to your endpoint. The git tool checks the cert and sees that it is properly expired and refuses to connect to an https connection with an invalid cert.

I think you are absolutely right 👍

**Comment by mamaretti** [comment]
Hi Dan, Hi Dr3,
thx for your replies. that could be the solution. 👍
Didn't thought, that it could be git version. 
I'm on Mojave and git-version is 2.21.1.  
Will try to update git an hope that will will fix it.

**Comment by DanPristupov** [comment]
@mamaretti the bundled git version in Fork is `2.31.1`. You can switch the instance in the application preferences.

**Comment by mamaretti** [comment]
Hi Dan.
I tried forked-bundled-git 2.31.1 and a newly system git version 2.33.0.

On both I still get "fatal: unable to access.... SSL certificate problem: certificate has expired"
...also when trying via comandline " git pull 'xxx' 'xxx'".

no plan what I'm doing wrong.
when I open the gitlabserver the SSL-certificate looks ok till December 2021.

so problem is not "Forks" i will continue to research. thanks for now

**Comment by mamaretti** [comment]
solved it by deleting "DST Root CA X3" certificate from /etc/ssl/cert.pem file 

Found here 
[https://stackoverflow.com/questions/69387175/git-for-windows-ssl-certificate-problem-certificate-has-expired#answer-69396425](https://stackoverflow.com/questions/69387175/git-for-windows-ssl-certificate-problem-certificate-has-expired#answer-69396425)

**Comment by dr3amnightmare** [comment]
@mamaretti  I figured it might be something like that. Glad to hear you're back up and running!
